api-specificatie/resolved/openapi.yaml

```diff
@@ -4023,7 +4054,7 @@ components:
       properties:
         href:
           type: string
-          example: https://datapunt.voorbeeldgemeente.nl/api/{major-versie}/resourcenaam?page=5
+          example: /resourcenaam?page=5
```

Comment:
Waarom staan hier ineens relatieve URI's? Betekent dit dat wij ook ineens relatieve paden moeten gaan invullen in de paginering links? Of mogen we ook gewoon absolute paden blijven teruggeven?
Als dit relatieve URI's moeten worden, dan moet dat middels een ticket worden afgestemd en afgehandeld omdat dit impact heeft op de implementatie aan de provider kand. Dit kan echt niet zomaar met een yaml wijziging in een common.yaml worden doorgevoerd!

Reply:
Deze relatieve URI's staan vanaf 22 dagen in versie 1.0.0 van common.yaml. Blijkbaar valt het nu op omdat ik de resolved versie opnieuw geresolved heb (en dat is de eerste keer tegen de v1.0.0 van de common.yaml aan)

Deze wijziging is overigens n.a.v. discussies over URI templating en relatieve URI's binnen BRK-bevragingen aangebracht. 
Blijkbaar hebben we dit onderwerp niet aangesneden binnen het BAG-project. 

Deze verrassingen wilden we juist voorkomen met het versioneren van de common.yaml. 
@fsamwel en @MelvLee Hoe gaan we hier mee om ? Het is natuurlijk maar een example, maar dit wekt wel de verwachting dat er een relatieve URL is.

Reply:
Bij BRK is inderdaad afgesproken om relatieve uri's te gebruiken voor interne endpoints.
De reden hiervoor is dat de response kleiner wordt en dat je als provider niet via een configuratie variabele de base url hoeft te beheren.
Het is echter geen verplichting omdat het voor consumers niet uitmaakt of hij absolute of relatieve urls krijgt.

Reply:
OK, dan geven wij voorlopig gewoon absolute paden terug.